languagetool-language-modules/ar/src/main/java/org/languagetool/language/Arabic.java

```diff
@@ -0,0 +1,198 @@
+/* LanguageTool, a natural language style checker
+ * Copyright (C) 2007 Daniel Naber (http://www.danielnaber.de)
+ * 
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301
+ * USA
+ */
+package org.languagetool.language;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.ResourceBundle;
+
+import org.languagetool.Language;
+import org.languagetool.LanguageMaintainedState;
+//import org.languagetool.chunking.Chunker;
+//import org.languagetool.chunking.ArabicChunker;
+import org.languagetool.languagemodel.LanguageModel;
+import org.languagetool.languagemodel.LuceneLanguageModel;
+import org.languagetool.rules.*;
+import org.languagetool.rules.ar.*;
+import org.languagetool.synthesis.Synthesizer;
+import org.languagetool.synthesis.ar.ArabicSynthesizer;
+import org.languagetool.tagging.Tagger;
+import org.languagetool.tagging.disambiguation.Disambiguator;
+import org.languagetool.tagging.disambiguation.rules.XmlRuleDisambiguator;
+import org.languagetool.tagging.ar.ArabicTagger;
+import org.languagetool.tokenizers.SRXSentenceTokenizer;
+import org.languagetool.tokenizers.SentenceTokenizer;
+import org.languagetool.tokenizers.WordTokenizer;
+import org.languagetool.tokenizers.ar.ArabicWordTokenizer;
+import org.languagetool.rules.spelling.hunspell.HunspellNoSuggestionRule;
+/**
+ * Support for Arabic - use the sub classes {@link BritishArabic}, {@link AmericanArabic},
+ * etc. if you need spell checking.
+ * Make sure to call {@link #close()} after using this (currently only relevant if you make
+ * use of {@link ArabicConfusionProbabilityRule}).
+ */
+public class Arabic extends Language implements AutoCloseable {
+
+  private Tagger tagger;
+  //private Chunker chunker;
+  private SentenceTokenizer sentenceTokenizer;
+  private Synthesizer synthesizer;
+  private Disambiguator disambiguator;
+  private WordTokenizer wordTokenizer;
+  private LuceneLanguageModel languageModel;
+
+  /**
+   * @deprecated use {@link AmericanArabic} or {@link BritishArabic} etc. instead -
```

Comment:
AmericanArabic and BritishArabic? There're no such variant most certainly.

It looks like a global substitute mistake s/English/Arabic/ which changed AmericanEnglish and BritishEnglish into AmericanArabic and BritishArabic.